catalog/ui/catalog-ui-search/src/main/webapp/component/search-settings/search-settings.less

```diff
@@ -31,14 +31,23 @@
     }
   }
 
-  > .editor-properties > button {
+  > .editor-properties > .editor-btn-group > button {
     display: none;
```

Comment:
We should group the base button properties here, things like width.

Reply:
@andrewkfiedler I'm seeing `display: none` to be the only properties shared across these buttons as I conditionally alter the width. Is there anything else I need to add here?